runtime/devnet/src/extensions/mod.rs

```diff
@@ -0,0 +1,1160 @@
+use codec::{Compact, Decode, Encode};
+use cumulus_pallet_parachain_system::RelaychainDataProvider;
+use frame_support::traits::{Contains, OriginTrait};
+use frame_support::{
+	dispatch::{GetDispatchInfo, RawOrigin},
+	pallet_prelude::*,
+	traits::{
+		fungibles::{approvals::Inspect as ApprovalInspect, metadata::Inspect as MetadataInspect},
+		nonfungibles_v2::Inspect as NonFungiblesInspect,
+	},
+};
+use pallet_contracts::chain_extension::{
+	BufInBufOutState, ChainExtension, Environment, Ext, InitState, RetVal,
+};
+use sp_core::crypto::UncheckedFrom;
+use sp_runtime::{
+	traits::{BlockNumberProvider, Dispatchable},
+	DispatchError, MultiAddress,
+};
+use sp_std::{boxed::Box, vec::Vec};
+use xcm::{
+	latest::{prelude::*, OriginKind::SovereignAccount},
+	VersionedXcm,
+};
+
+use crate::{
```

Comment:
Join usings with above block as standard.

Reply:
As for our coding standards we are separating internal and external crate imports. If this is what you meant.